packages/mui-material/src/SnackbarContent/SnackbarContent.js

```diff
@@ -31,7 +31,7 @@ const SnackbarContentRoot = styled(Paper, {
   return {
     ...theme.typography.body2,
     color: theme.vars
-      ? theme.vars.palette.text.primary
+      ? theme.vars.palette.SnackbarContent.text
```

Comment:
To make the variables consistent with the others, I think it should be `color`.
```suggestion
      ? theme.vars.palette.SnackbarContent.color
```